README.md

```diff
@@ -266,7 +266,7 @@ airbrake.Notify(notice, nil)
 
 ### Performance Monitoring
 
-You can read more about our [Performance Monitoring offering in our docs](docs/performance).
+You can read more about our [Performance Monitoring offering in our docs][docs/performance].
```

Comment:
This link 404's for me. Does it work for you?

Reply:
Yes, it works when I preview the file in GitHub. Are you testing it on GitHub?

Reply:
It works now for me as well. Sry, I'm not sure what happened before :)